### ISSUE TITLE
AEM6.1 with SP1 synthetic include not working

### ISSUE THREAD

**Issue by vivekm2017** [wttech/Sling-Dynamic-Include#20]
Hi, 

I have integrated SDI in my project but finding issues with synthetic include. When the node exist, include is working fine but when the node does not exist, the include does not work based on resource type.

I am getting following error-

unable to include "/content/proj/en/produce/title-of-teaser/placement/level3/_jcr_content/inlineEdit.wkd.nocache.html//apps/proj/components/content/inlineeditingtoolbar" in parsed file C:/Apache22/cache/content/proj/en/produce/title-of-teaser/placement/level3.wkd.html

Tried other way around also-
unable to include "/content/proj/en/produce/title-of-teaser/placement/level3/_jcr_content/inlineEdit.wkd.nocache.html/proj/components/content/inlineeditingtoolbar" in parsed file C:/Apache22/cache/content/proj/en/produce/title-of-teaser/placement/level3.wkd.html

The AEM version is 6.1 with SP1. I have tried changing service ranking for synthetic resource to -2400 (>-2500) and -2600 (<-2500). But no luck.

Can you please suggest what might be wrong.

**Comment by pacoolsky** [comment]
ON AEM 6.1 left the ranking as it is by default (Integer.MIN) and it should work.

"Unable to include" - it looks more like a issue with the SSI on Apache itself (configuration or permisssions). I suggest to try the get the 2nd url directly against AEM/CQ instance to check if SDI works ok.

**Comment by jaspians** [comment]
I was able to resolve the "Unable to include" issue (i don't remember now ;) what I changed) but in case of synthetic resource dynamic variables are not resolved. Let me explain it- suppose I had a component "inlineeditingtoolbar" for inclusion as synthetic resource (I mean this node does not exist) then if inlineeditingtoolbar.jsp/ inlineeditingtoolbar.html has any variable that gets value from design dialog then the variable value is not available. If it has just static HTML/ text then it worked fine.
